app/components/metrics_summary_component/_index.scss

```diff
@@ -0,0 +1,35 @@
+@use "pkg:govuk-frontend" as govuk;
+
+.app-metrics__container {
+  margin-bottom: govuk.govuk-spacing(6);
+
+  @include govuk.govuk-media-query(tablet) {
+    display: grid;
+    grid-template-columns: repeat(3, minmax(govuk.govuk-px-to-rem(260), 6fr));
```

Comment:
I'm seeing some wrapping issues at some screen sizes - these can be resolved by using the original minmax values from the admin component:
```suggestion
    grid-template-columns: repeat(3, minmax(govuk.govuk-px-to-rem(100), 1fr));
```

Reply:
![image](https://github.com/user-attachments/assets/e5feb8e0-7d2a-4512-8c6b-2c46ec48864f)